package.json

```diff
@@ -64,13 +64,13 @@
     "yargs-parser": "^20.0.0"
   },
   "devDependencies": {
+    "@fastify/autoload": "^4.0.0",
```

Comment:
`@fastify/autoload` should be bumped to v5.0.0

Reply:
Sure! Done [a294e1c](https://github.com/fastify/fastify-cli/pull/490/commits/a294e1ca4e9f836ccda168ac197aa816693891e3)